tests/phpunit/bootstrap.php

```diff
@@ -18,6 +18,12 @@
 
 $loader->register();
 
+// Make CRM_Civioffice_ExtensionUtil available.
+require_once __DIR__ . '/../../civioffice.civix.php';
+
+// Ensure function ts() is available - it's declared in the same file as CRM_Core_I18n.
+\CRM_Core_I18n::singleton();
```

Comment:
This is not the case anymore with CiviCRM `5.74`+, see civicrm/civicrm-core@b09cee8ecac6108801aa697f8c921421d92de0bf

Reply:
Replaced by
```php
if (!function_exists('ts')) {
  // Ensure function ts() is available - it's declared in the same file as CRM_Core_I18n in CiviCRM < 5.74.
  // In later versions the function is registered following the composer conventions.
  \CRM_Core_I18n::singleton();
}
```